alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/resources/libreoffice_engine_config.json

```diff
@@ -227,8 +227,6 @@
 
         {"sourceMediaType": "application/vnd.ms-excel.template.macroenabled.12",                                                        "priority": 110, "targetMediaType": "application/pdf" },
 
-        {"sourceMediaType": "application/vnd.ms-excel.addin.macroenabled.12",                                                           "priority": 110, "targetMediaType": "application/pdf" },
```

Comment:
We seem to be commenting-out existing config ... that was delivered in T-Engines 2.2.2 ?

Does this means we have 2x (of the x39 transforms) that do not work - see also other comment ? 

What's the plan for these transforms:

- fixable (add test)
- requires further investigation (raise tech debt)
- obsolete (no plans to support) - also check with @alandavis

Reply:
Yes, the above comment is correct. While fixing failing L/O tests, I discovered that the removed transform above isn't working at all. I believe Ayman discovered one which isn't working in I/M as well.

Please refer to https://github.com/Alfresco/alfresco-transform-core/pull/252/#discussion_r437327450 this comment for more info.

Reply:
Thanks :-)